.github/ISSUE_TEMPLATE/01-bug-report.yml

```diff
@@ -33,12 +33,7 @@ body:
   - type: textarea
     attributes:
       label: System information
-      description: Provide output of the command `termux-info`, enclosed it in a Markdown **code block** (\`\`\` ... \`\`\`).
-      value: |
-        termux-info:
-
-        ```
-
-        ```
+      description: Provide output of the command `termux-info`.
+      render: shell
```

Comment:
I assume this puts it in a code block automatically?

Reply:
Yeah. So there will be no automatic markdown rendering in the case if user removes backtics and pastes `termux-info` output to textarea.